source/extensions/filters/http/proto_message_extraction/extraction_util/extraction_util.cc

```diff
@@ -125,8 +126,14 @@ void GetMonitoredResourceLabels(absl::string_view label_extractor,
 }
 
 WireFormatLite::WireType getWireType(const Field& field_desc) {
+  if (field_desc.kind() == Field::TYPE_UNKNOWN) {
+    // Return an invalid wire type for unknown field types
+    // Using max value to indicate invalid rather than -1
+    return static_cast<WireFormatLite::WireType>(std::numeric_limits<int>::max());
+  }
+
   static WireFormatLite::WireType field_kind_to_wire_type[] = {
-      static_cast<WireFormatLite::WireType>(-1), // TYPE_UNKNOWN
+      WireFormatLite::WIRETYPE_VARINT,           // Placeholder for TYPE_UNKNOWN (index 0)
```

Comment:
This change should probably just be
```
static_cast<WireFormatLite::WireType>(std::numeric_limits<WireFormatLite::WireType>::max()),
```
(I don't know if that will work, but there must be *something* that would work and keep it to a one-liner without having an inefficient early special-case branch for the case that should never happen that probably results in a coverage dip.)